src/encoded/loadxl.py

```diff
@@ -593,7 +593,7 @@ def get_pipeline(testapp, docsdir, test_only, item_type, phase=None, method=None
         remove_keys('spikeins_used', 'adapters'),
     ],
     'experiment': [
-        remove_keys('possible_controls', 'related_files', 'supersedes'),
+        remove_keys('possible_controls', 'related_files', 'supersedes', 'analyses'),
```

Comment:
Oh, I would not have expected you needed this, but I guess it does look down and see linkTos and those files have to be already loaded in PG.

Reply:
Yes. I tried. Will fail and get corresponding errors without it:
```
ERROR:encoded:VALIDATION FAILED: [{'description': "'ENCFF003MRN' not found", 'location': 'body', 'name': ['analyses', 0, 0]}, {'description': "'ENCFF005MRN' not found", 'location': 'body', 'name': ['analyses', 0, 1]}, {'description': "'ENCFF004MRN' not found", 'location': 'body', 'name': ['analyses', 1, 0]}, {'description': "'ENCFF006MRN' not found", 'location': 'body', 'name': ['analyses', 1, 1]}, {'description': "'ENCFF807TRA' not found", 'location': 'body', 'name': ['analyses', 1, 2]}, {'description': "'ENCFF783YZT' not found", 'location': 'body', 'name': ['analyses', 1, 3]}]
ERROR:encoded:experiment row 48: 422 Unprocessable Entity (/experiment)
{'accession': 'ENCSR123MRN', 'uuid': 'e828ec88-8f74-437f-8bf7-8535ee34819e', 'biosample_ontology': 'tissue_UBERON_0001891', 'date_released': '2015-08-31', 'status': 'released', 'lab': '/labs/john-stamatoyannopoulos/', 'assay_term_name': 'microRNA-seq', 'submitted_by': '/users/4c23ec32-c7c8-4ac0-affb-04befcc881d4/', 'analyses': [['ENCFF003MRN', 'ENCFF005MRN'], ['ENCFF004MRN', 'ENCFF006MRN', 'ENCFF807TRA', 'ENCFF783YZT']], 'date_submitted': '2015-04-20', 'award': 'U54HG007010'}

INFO:encoded:Loaded 51 of 52 experiment (phase 1). CREATED: 51, UPDATED: 0, SKIPPED: 0, ERRORS: 1

```